parsons/zoom/zoom.py

```diff
@@ -23,15 +26,49 @@ class Zoom:
             variable set.
     """
 
-    def __init__(self, api_key=None, api_secret=None):
+    def __init__(self, account_id=None, client_id=None, client_secret=None):
+        self.account_id = check_env.check("ZOOM_ACCOUNT_ID", account_id)
```

Comment:
Could you:
1. Rebase your branch on the major-release branch
2. Change the PR to target major-release
3. Add details in the PR description for all of the changes that would break existing code? In this case, I think it's just the changes to the env vars. This is really important so users know what they need to migrate when they upgrade.

Reply:
#3 is kind of mentioned, but it would be nice to list what the ENV variables were before and what the exact ENV variables need to be now.